templates/documents/general_upload.html

```diff
@@ -10,8 +10,29 @@
 * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
 *}
 
+<script>
+    function checkFileSize() {
+        var fileInput = document.querySelector('input[type="file"]');
+        var totalSize = 0;
+
+        // Get the size of the selected file
+        for (var i = 0; i < fileInput.files.length; i++) {
+            totalSize += fileInput.files[i].size;
+        }
```

Comment:
The MAX_FILE_SIZE is the maximum allowed size for each individual file, not the combined total size of all files.
So this verification is invalid for our purposes.